src/_component/recommendations/recommendations.pug

```diff
@@ -0,0 +1,20 @@
+#recomamendations
+- var sectionTitle = data.recommendations.sectionTitle
```

Comment:
Am I insane or does this not get used anywhere?
Also, love the Id :D #Recomamendations

Reply:
Ah I see what is happening, it gets used by the following inclusion of `../sectiunea-titlu/sectiunea-titlu`